release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
+* [alpha] Add limited support for pod "checkpointing" in the kubelet to help enable self-hosting. ([#378](https://github.com/kubernetes/features/issues/378), [@timothysc](https://github.com/timothysc))
+
+
+### **Cluster Lifecycle**
+#### kubeadm
+* [beta] A new `phase` subcommand supports performing only subtasks of the full `kubeadm init` flow. Combined with fine-grained configuration, kubeadm is now more easily consumable by higher-level provisioning tools like kops or GKE. ([#356](https://github.com/kubernetes/features/issues/356), [@luxas](https://github.com/luxas), [@justinsb](https://github.com/justinsb))
+
+* [alpha] A new `upgrade` subcommand allows you to automatically upgrade a self-hosted cluster created with kubeadm. ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
```

Comment:
Should be [beta]. Not sure whether self-hosted will be true or not, but I'm fine with leaving that as-is, as I think we're gonna meet the deadline for self-hosting upgrades as well (add-first-then-kill)